db/migrations/20161026_1148_add_secondary_index_to_incidents.rb

```diff
@@ -0,0 +1,50 @@
+# Migration to add a secondary index to the Incidents table.
+class AddSecondaryIndexToIncidents < DynamoDB::Migration::Unit
+  def update
+    logger = Logger.new(STDOUT)
+
+    logger.info "Running migration #{self.class.name}:"
+
+    client.update_table(
+      table_name: "#{Dynamoid.config.namespace}_incidents",
+      attribute_definitions: [
+        {
+          attribute_name: "id",
+          attribute_type: "S"
+        },
+        {
+          attribute_name: "ursus_id_str",
+          attribute_type: "S"
+        }
+      ],
```

Comment:
Why do we need to update these attribute definitions and the provisioned throughput below?